pkg/controller/networkpolicy/external_entity.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package networkpolicy provides NetworkPolicyController implementation to manage
+// and synchronize the Pods and Namespaces affected by Network Policies and enforce
+// their rules.
+
+package networkpolicy
+
+import (
+	"reflect"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/apis/core/v1alpha1"
+	"github.com/vmware-tanzu/antrea/pkg/apis/core/v1alpha2"
+)
+
+// externalEntityV1Alpha1ToAlpha2 converts a v1alpha1.ExternalEntity object to a
+// v1alpha2.ExternalEntity object.
+func externalEntityV1Alpha1ToAlpha2(ee *v1alpha1.ExternalEntity) *v1alpha2.ExternalEntity {
```

Comment:
typically i see conversions in a conversion.go under pkg/apis/... , however i think it might okay in this case since we intend to not support the older version for long time